src/index.html

```diff
@@ -11,12 +12,110 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
```

Comment:
The `crossorigin` attribute value is incorrect. It should be either `anonymous` or `use-credentials` according to the task requirements.